whitespace_checks.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
Please don't use `/bin/bash` in scripts. Use `/usr/bin/env bash`, it's more portable, e.g. works on nixos and rare non-linux unixes.